src/components/cody/dual-theme/CodyIntroDualTheme.tsx

```diff
@@ -26,62 +28,69 @@ export const CodyIntroDualTheme: FunctionComponent<CodyIntroDualThemeProps> = ({
     description,
     titleSize,
     descriptionSize,
+    isVariant = false,
 }) => (
     <ContentSection
         parentClassName="!py-0 !px-0"
-        className={classNames('-mt-8 pt-0 text-center md:mt-0', wrapperClassName, {
-            'md:pt-[10px]': isLight,
-            'md:pt-[22px]': !isLight,
-        })}
+        className={classNames('pt-16 md:mt-0 md:pt-[123px]', wrapperClassName)}
     >
-        <div className="mx-auto w-full px-6 md:w-[849px] lg:w-[895px]">
-            <h1
+        <h1
+            className={classNames('mx-auto w-full font-semibold', titleSize || 'text-4xl sm:text-6xl', {
+                'text-white': !isLight,
+                'text-gray-700': isLight,
+            })}
+        >
+            {(!isVariant && title) ?? <span>Code more, type&nbsp;less</span>}
+
+            {isVariant && (
+                <>
+                    <span className="hidden md:block">{title}</span>
```

Comment:
The mobile breakpoint should have the same copy as the larger breakpoints. Apologies if the mockup was misleading!